### PR TITLE
Update terraform-provider-rke to v1.3.4

### LINKED ISSUE #38: Upgrade terraform-provider-rke to v1.3.4
Release details: https://github.com/rancher/terraform-provider-rke/releases/tag/v1.3.4

### PR DESCRIPTION
**Pull request by viveklak**
Fixes #38. 

Also switches to the upstream provider directly instead of using the fork. The only real change in the fork is https://github.com/rancher/terraform-provider-rke/commit/81bf6b33632124463163e8dada32b6ddb17391ba which doesn't seem worth maintaining a fork for.

### CODE REVIEW

**Review by jazzyfresh — APPROVED** [review]
LGTM

I encountered a bug updating tfbridge, so let's just merge this and do package updates in another branch.

File: examples/cluster/python/__main__.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def wait_for_ready(private_ip):
     # time to spin up and be ready
     # if we had a custom image with docker installed then it would
     # be a faster start up time
-    time.sleep(120)
+    time.sleep(240)
```
Comment (jazzyfresh) [inline examples/cluster/python/__main__.py:98]: creating a thread here to debug nodejs test failure i'm seeing [here](https://github.com/pulumi/pulumi-rke/actions/runs/4070063399/jobs/7010589966#step:22:413)
> Failed running cluster err:[workerPlane] Failed to bring up Worker Plane: [Failed to verify healthcheck: Failed to check http://localhost:10248/healthz for service [kubelet] on host [34.221.48.176]: Get "http://localhost:10248/healthz": Unable to access the service on localhost:10248. The service might be still starting up. Error: ssh: rejected: connect failed (Connection refused), log: Error: failed to run Kubelet: unable to determine runtime API version: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial unix: missing address"]

cc @aq17 @AaronFriel

**Comment by folliehiyuki** [comment]
Hi! I would love to see this Pulumi provider upgrades the Terraform provider dependency to 1.4.2. The current version (1.2.2) only supports K8s versions in between 1.17 and 1.20.

**Comment by lukehoban** [comment]
Picked this up in #125 which replaces this.

**Pull request closed without merging**